topics/sequence-analysis/tutorials/ncbi-blast-against-the-madland/tutorial.md

```diff
@@ -0,0 +1,160 @@
+---
+layout: tutorial_hands_on
+
+title: "NCBI BLAST+ against the MAdLand"
+zenodo_link: 'https://zenodo.org/record/4710649'
+questions:
+- "How to access Galaxy server?"
+- "How can we perform Blast analysis on Galaxy?"
+- "What is MAdLand DB?"
+
+objectives:
+- "Load fasta sequence into Galaxy"
+- "Perform NCBI-Blast+ analysis on Galaxy"
+
+time_estimation: "20m"
+key_points:
+- Blast tool searches a database of sequences for similar sequences to a query sequence.
+- MAdLand is a database of fully sequenced plant and algal genomes, with an emphasis on non-seed plants and streptophyte algae that can be use for sequence similarity search.
+
+contributors:
+- deeptivarshney
+
+---
+
+# Introduction
+
+<!-- This is a comment. -->
+
+MAdLand is a collection of fully sequenced plant and algal genomes, with a focus on non-seed plants and streptophyte algae. For comparison, it includes genomes from fungi, animals, the SAR group, bacteria, and archaea. It is developed and maintained by the [Rensing lab](http://plantco.de). The species are abbreviated by a 5 letter code, which consists of the first three letters of the genus and the first two of the species name, e.g. CHABR for Chara braunii. We add the gene ID to that and additional shortcuts, like whether it is plastome encoded (pt) or transcriptome-based (tr, in cases when no genome is available yet).
```

Comment:
I cannot find any references about that database in this link http://plantco.de. Is there any publication that could be referenced in order to find more information?